ProblemR.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+ 
+using namespace std;
+ 
+int main() {
+    long long N, y, m, d;
+    
+    cin >> N;
+    
+    y = N / 365;
+    m = (N - (365 * y)) / 30;
+    d = (N - (365 * y)) - (m * 30);
```

Comment:
Extracting constants like 365 to avoid repetition.

(Yup, totally minor & acceptable for problem solving)